components/org.wso2.carbon.identity.conditional.auth.functions.notification/pom.xml

```diff
@@ -94,6 +94,11 @@
             <groupId>org.wso2.carbon.identity.framework</groupId>
             <artifactId>org.wso2.carbon.identity.central.log.mgt</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.openjdk.nashorn</groupId>
+            <artifactId>nashorn-core</artifactId>
+            <scope>provided</scope>
```

Comment:
Shall we add the scope to test as this is used only in a testcase

Reply:
Fixed by - https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/216/commits/838a3e5068ed65d1b6557e26e207a1818eb35ad1